bangazonapi/models/product.py

```diff
@@ -14,7 +14,7 @@ class Product(SafeDeleteModel):
     customer = models.ForeignKey(
         Customer, on_delete=models.DO_NOTHING, related_name='products')
     price = models.FloatField(
-        validators=[MinValueValidator(0.00), MaxValueValidator(10000.00)],)
+        validators=[MinValueValidator(0.00), MaxValueValidator(17500.00)],)
```

Comment:
Have you tested to make sure that validation is enforced when you enter a new product with price over max?

Search `evening-cohort-12` for `ModelForm` for a helpful thread about this ticket.